src/content_scripts/profilePage.js

```diff
@@ -22,13 +22,16 @@ function createFollowButton() {
   username = username.split('#')[0];
   username = username.split('?')[0];
 
-  $(document).ready(function () {
-      if(window.location.href.indexOf("profile") > -1) {
-        $followButton = $('<button></button>').attr({ id: 'se_follow_button' }).css("margin-top", "6px");
-      } else {
-        $followButton = $('<button></button>').attr({ id: 'se_follow_button' }).css("bottom", "5px");
-      }
-  });
+  function makeStyledButton(profileElement, profileValue, generalElement, generalValue) {
+    if($('#tabs').find("a:first").hasClass("youarehere")) {
+      console.log($('#tabs').children("a:first").hasClass("youarehere"));
+      $followButton = $('<button></button>').attr({ id: 'se_follow_button' }).css(profileElement, profileValue);
+    } else {
+      $followButton = $('<button></button>').attr({ id: 'se_follow_button' }).css(generalElement, generalValue);
+    }
+  }
+
+  makeStyledButton("margin-top", "6px", "bottom", "5px");
```

Comment:
Signatures and Invocation does not match.

> function makeStyledButton(profileElement, profileValue, generalElement, generalValue)
>  makeStyledButton("margin-top", "6px", "bottom", "5px");


It looks like `makeStyledButton`  will create a styled button with given styles. I see the following problems with this approach:

1. Signature says first argument is some element i.e. `profileElement` while you treat it as some style `margin`
2. This method is not generic enough which could be used elsewhere. Rather I would have something like this

    createElement(elementName, cssProperties, attributes)

And use it like this:

    var$ buttonEl = createElement('button', { 'margin': '1px' }, { id: 'se_follow_button' });

Now you can consume this button elsewhere in your code. But this will be like creating a thin layer over jquery apis but ok to have.

Thoughts ?